SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -477,7 +443,6 @@ const TMap<Worker_ComponentId, FString> ServerAuthorityWellKnownComponents = {
 	{ ALWAYS_RELEVANT_COMPONENT_ID, "unreal.AlwaysRelevant" },
 	{ DORMANT_COMPONENT_ID, "unreal.Dormant" },
 	{ VISIBLE_COMPONENT_ID, "unreal.Visible" },
-	{ SERVER_RPC_ENDPOINT_COMPONENT_ID_LEGACY, "unreal.UnrealServerRPCEndpointLegacy" },
```

Comment:
Delete the well known schema also, will also require updating the tests :(

Reply:
The file still remains! Has the server->server rpcs in it still